victor/src/layout/mod.rs

```diff
@@ -169,8 +168,10 @@ impl BlockLevelBox {
                     contents,
                 )
             }
-            BlockLevelBox::AbsolutelyPositionedBox(absolutely_positioned_box) => {
-                absolutely_positioned_box.layout(tree_rank, absolutely_positioned_fragments)
+            BlockLevelBox::AbsolutelyPositionedBox(box_) => {
+                let fragment = Fragment::Box(BoxFragment::zero_sized(box_.style.clone()));
+                absolutely_positioned_fragments.push(box_.layout(Vec2::zero(), tree_rank));
+                fragment
```

Comment:
I think this looks wrong. There should be no fragment at all for the hypothetical box, not a box fragment with zero child fragments (which can still paint borders, etc).